source/controllers/disk.py

```diff
@@ -162,7 +162,8 @@ def sync_disks():
                 try:
                     disk = DiskList.get_by_alias(disk_info['aliases'][0])
                     if disk_name != disk.name:
-                        DiskController._logger.info('Detected moved disks, changing disk names to circumvent unique constraints')
+                        DiskController._logger.info('Detected with alias{0} its name has changed from {1} to {2}, \
```

Comment:
Let's make it even more clear? 'Disk with alias {0}'s name has changed... instead of the 'Detected..'